homeassistant/components/notify/xmpp.py

```diff
@@ -59,11 +65,11 @@ def __init__(self):
 
             logging.basicConfig(level=logging.ERROR)
 
-            self.use_tls = True
+            self.use_tls = use_tls
             self.use_ipv6 = False
             self.add_event_handler('failed_auth', self.check_credentials)
             self.add_event_handler('session_start', self.start)
-            self.connect()
+            self.connect(use_tls=self.use_tls, use_ssl=False)
```

Comment:
I think all forms of SSL have by now been deprecated.

On April 9, 2016 6:46:56 PM EDT, Paulus Schoutsen <notifications@github.com> wrote:
>>              self.use_ipv6 = False
>>              self.add_event_handler('failed_auth',
>self.check_credentials)
>>              self.add_event_handler('session_start', self.start)
>> -            self.connect()
>> +            self.connect(use_tls=self.use_tls, use_ssl=False)
>
>Are we always not going to use ssl ? 
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/balloob/home-assistant/pull/1763/files/33e87c2d24494382762ac8f2d8b85d74bf7f687a#r59122965